readme.md

```diff
@@ -0,0 +1,30 @@
+![Polylang](assets/polylang-logo.png)
+
+Welcome to the Polylang repository on GitHub. Here you can browse the source, look at open
+issues and keep track of development.
```

Comment:
Strange this is spread on 2 lines (but seems to work as expected anyway). I suggest: 
- **discuss** open issues instead of **look at** open issues.
-  and keep track of **the** development.